website/server/models/group.js

```diff
@@ -623,11 +642,20 @@ schema.methods.finishQuest = async function finishQuest (quest) {
     }
   });
 
-  let q = this._id === TAVERN_ID ? {} : {_id: {$in: this.getParticipatingQuestMembers()}};
+  let participants = this.getParticipatingQuestMembers();
```

Comment:
That's why I left the ternary operator in the beginning because I wasn't sure of the impact of calling this with a tavern quest. The getParticipatingMembers method depends on this.quest.members which I'm not sure the state of for tavern quests as I don't know how they are started or populated.

I can add the ternary operation back or make the getParticipatingQuestMembers method explicitly handle tavern quests to return an empty object. What do you think?